### DISCUSSION TITLE
Add Thailand language

### DISCUSSION THREAD

**Discussion by ATOMONLINE** [AzuraCast/AzuraCast#5205] [General]
Any chance to add the Thailand language? I am ready to do the translate :)

**Comment by BusterNeece** [comment]
@ATOMONLINE I've added Thai as a supported language in Crowdin. Please let me know when translations have been submitted so we can incorporate them into the application.

**Reply by ATOMONLINE** [reply]
@SlvrEagle23 How can I submit translations?
https://crowdin.com/project/azuracast-thai/invite?d=055585l6s4j5e5c373b3o463p4

**Reply by Vaalyn** [reply]
When you created a Crowdin account and joined the translation project for the Thai language you can go here and start submitting translations: https://crowdin.com/translate/azuracast-thai/26/en-th

**Reply by ATOMONLINE** [reply]
> @ATOMONLINE I've added Thai as a supported language in Crowdin. Please let me know when translations have been submitted so we can incorporate them into the application.

@SlvrEagle23 Where can I send translation files?

**Reply by BusterNeece** [reply]
@ATOMONLINE Translations are managed at https://translate.azuracast.com

**Reply by ATOMONLINE** [reply]
> @ATOMONLINEการแปลได้รับการจัดการที่https://translate.azuracast.com

I will send translation but I can't see my language.
